native/core/src/execution/expressions/bloom_filter_might_contain.rs

```diff
@@ -140,4 +141,8 @@ impl PhysicalExpr for BloomFilterMightContain {
             Arc::clone(&children[1]),
         )?))
     }
+
+    fn fmt_sql(&self, _: &mut Formatter<'_>) -> std::fmt::Result {
+        unimplemented!()
+    }
```

Comment:
DataFusion now supports a new EXPLAIN output, but this is not exposed in Comet, so we do not need to implement these new methods.